src/views/tiles/NodeView.js

```diff
@@ -146,6 +147,15 @@ exports = Class(ImageView, function (supr) {
 								y: this.style.y + friends.y * tileSettings.tileHeight - friendsTop + friendsBottom
 							});
 						}
+					} else if (this._itemCtors[tag] instanceof Viewpool) {
+							itemView = itemViews[tag] = this._itemCtors[tag].obtainView({
+								superview: this.getSuperview(),
+								tag: tag,
+								zIndex: 999999999,
+								tile: tile,
+								x: this.style.x,
+								y: this.style.y
+							});
```

Comment:
show/hide should happen here. either override show function of instance or add new function update , which will get invoked from here and update function will take care of handling rendering.